config/settings.py

```diff
@@ -155,6 +155,8 @@
     "include_jquery": True,
 }
 
+SIMILARITY_RATIO = 0.5
```

Comment:
We should add a comment on what this variable is, why it is needed, and why it has this default value